RearrangeScreensModule/src/main/java/com/siri/RearrangeScreensSDK/ScreenUtils.kt

```diff
@@ -0,0 +1,13 @@
+package com.siri.RearrangeScreensSDK
+
+import android.content.Context
+import android.widget.Toast
+
+object ScreenUtils {
+
+    fun takeScreenNames(context : Context?, stringNames:String? = null)
```

Comment:
Four nitpicks:

Dose the `stringNames` parameter accept only one name or many?  If just one screen name (and it's not an array here, so I'm guessing it's just one), then why plural?  Your method name and your parameter name is plural, but seems it's just to specify one screen name?

Why is the word `string` in the name of the variable?  We know it's a string from its type.  Why not simply `name` or `screenName`?

Why is `stringNames` null by default?  What would it mean to not set it to anything? 

Also, a nitpick, but... you might want to install a linter because your formatting seems inconsistent:

```
context : Context?
```
but
```
stringNames:String?
```
Which convention do you want to follow?  Spaces after/before a colon or no spaces at all?  Some options:
```
context:Context
```
or
```
context: Context
```
or 
```
context : Context
```

Reply:
I addressed other parts of this comment,  coming to linter part, I will take it up next sprint